circe-akka-serializer/src/main/resources/reference.conf

```diff
@@ -1,5 +1,19 @@
 # Default configuration
 
 org.virtuslab.ash {
+
   verbose-debug-logging = off
+
+  # settings for compression of the payload
+  compression {
```

Comment:
I'd extract a sub-section for `circe`, to match package structure in Scala code

Reply:
done